app/styleguide/page.tsx

```diff
@@ -1037,23 +1059,161 @@ function Page() {
 
                 <div><pre><code>
                   {`.grid-view {
-                    --_gap: var(--grid-list-grid-gap, 1rem);
+    --_gap: var(--grid-list-grid-gap, 1rem);
 
-                    display: grid;
+    display: grid;
 
-                    padding: var(--_gap);
-                    grid-gap: var(--_gap);
-                    grid-template-columns: repeat(4, 1fr);
-                    grid-template-rows: auto;
-                  }
-                `}
-                </code></pre>
-                </div>
+    padding: var(--_gap);
+    grid-gap: var(--_gap);
+    grid-template-columns: repeat(4, 1fr);
+    grid-template-rows: auto;
+  }
+`}
+                </code></pre></div>
               </li>
             </ul>
 
-            <h2>Layout</h2>
-            <p>TBD ...</p>
+            <h2><code>Layout Container</code> (Base)</h2>
+            <p>The stanard <code>{`<LayoutContainer>`}</code> wraps content containers to provide
+            some common container within the layout container.</p>
+            <LayoutContainer style={{'--layout-container-background': 'var(--slate-200)'}}>
+              <ContentContainer>
+                <div><pre><code>
+                  {`<LayoutContainer style={{'--layout-container-background': 'var(--slate-100)'}}>
+  <ContentContainer> ... </ContentContainer>
+</LayoutContainer>
+`}
+                </code></pre></div>
+              </ContentContainer>
+            </LayoutContainer>
+
+            <h3>CSS Variables</h3>
+            <dl>
+              <dt><code>--layout-container-padding</code></dt>
+              <dd>
+                Set the container padding, defaults to: <code>0</code>
+              </dd>
+
+              <dt><code>--layout-container-background</code></dt>
+              <dd>
+                Set the container background, defaults to: <code>transparent</code>
+              </dd>
+            </dl>
+
+            <h2><code>Content Container</code> (Base)</h2>
+            <ContentContainer style={{'border': '1px solid var(--slate-600)'}}>
+              <p>This conteint is wrapped in the following container.</p>
+              <p><strong>Note</strong> that the style property is not required
+                to use the container. It's only here to show that you can use it
+                this way, and to show the outline here in the styleguide.</p>
+              <div><pre><code>{`<ContentContainer style={{'border': '1px solid var(--purple-600)'}}>
```

Comment:
L1109: Ideally we will not be using inline styles, and placing all styles in corresponding CSS Modules in the same directory.

Reply:
@jupiter007 The only reason I have the inline styles here is just to show the CSS variables in use directly on the styleguide, and so that I can quickly change the colours in-place. I wouldn't use inline styles myself, but it's a quick, handy override in non-critical places like the Styleguide.